pixi.toml

```diff
@@ -174,25 +174,27 @@ MOZ_HEADLESS = "1"
 PANEL_IPYWIDGET = "1"
 
 [feature.doc.dependencies]
+firefox = "*"
+geckodriver = "*"
 lxml = "*"
 nbsite = ">=0.8.4"
 selenium = "*"
 
 [feature.doc.tasks]
-_docs-refmanual = 'python ./doc/generate_modules.py panel -d ./doc/api -n panel -e tests'
-_docs-convert-gallery = 'python scripts/gallery/convert_gallery.py'
-_docs-generate = 'nbsite build --what=html --output=builtdocs --org holoviz --project-name panel'
-_docs-copy-panel-dist = 'cp -r ./panel/dist ./builtdocs/panel_dist'
-_docs-pyodide = 'panel convert examples/gallery/*.ipynb doc/how_to/*/examples/*.md --to pyodide-worker --out ./builtdocs/pyodide/ --pwa --index --requirements doc/pyodide_dependencies.json'
-docs-server = 'python -m http.server 5500 --directory ./builtdocs'
-
-[feature.doc.tasks.docs-build]
+_docs-build-api-reference = 'python ./doc/generate_modules.py panel -d ./doc/api -n panel -e tests'
+_docs-build-app-gallery = 'python scripts/gallery/convert_gallery.py'
+_docs-build-main = 'nbsite build --what=html --output=builtdocs --org holoviz --project-name panel'
+_docs-copy-results = 'cp -r ./panel/dist ./builtdocs/panel_dist'
+_docs-convert-examples-to-pyodide = 'panel convert examples/gallery/*.ipynb doc/how_to/*/examples/*.md --to pyodide-worker --out ./builtdocs/pyodide/ --pwa --index --requirements doc/pyodide_dependencies.json'
+serve-docs = 'python -m http.server 5500 --directory ./builtdocs'
+
+[feature.doc.tasks.build-docs]
```

Comment:
I wholeheartedly disagree with this change. 

The naming scheme of the tasks of `first-second` is like a TOC, where `first` is the chapter and `second` is the sub-chapters.  

This task is related to documentation; therefore, its first name is `docs`. If you want to focus on documentation, you should be able to look at the command starting with `docs-`. The tasks starting with `build-` are about building packages. 

It is unfortunate that `build-*` and `docs-build` both have `build` in the name. Maybe we should call them `docs-make` or something else to make it clearer that they are different. 

FWIW: The reason I chose `test`, `docs`, and `build` is that that is what the .github workflow file is called, so it's not the best reason.

Reply:
I'm okay with `docs-make`